src/Visualization/Python/GenerateXdmf.py

```diff
@@ -23,7 +23,7 @@ def generate_xdmf(file_prefix, output, subfile_name, start_time, stop_time,
                         "python 3 instead.")
 
     h5files = [(h5py.File(filename, 'r'), filename)
-               for filename in glob.glob(file_prefix + "*.h5")]
+               for filename in glob.glob(file_prefix + "[0-9]*.h5")]
```

Comment:
This will match `VolumeData0test.h5` but not `VolumeData.h5`. Was that your intention?

Reply:
Yes, precisely what I want :) It used to also match `VolumeDataBlah0.h5` which meant I got some _really_ confusing visualizations at some points :P

Reply:
Hehe ok :D